tests/integration/sdk_api/projects/test_project.py

```diff
@@ -107,7 +107,7 @@ def test_overwrite_project(self):
                 project=project.metadata.name,
             )
 
-        projects = db.list_projects()
+        projects = db.list_projects(format_=mlrun.common.schemas.ProjectsFormat.full)
         assert len(projects) == 1
         assert projects[0].metadata.name == project_name
```

Comment:
```suggestion
        projects = db.list_projects()
        assert len(projects) == 1
        assert projects[0] == project_name
```

same for other changes